tests/chart_tests/test_probes.py

```diff
@@ -1,40 +1,82 @@
+    "charts/alertmanager/templates/alertmanager-statefulset.yaml": {
+        "alertmanager": {**default_probes, "authProxy": default_probes}
+    },
+    "charts/astronomer/templates/astro-ui/astro-ui-deployment.yaml": {"astronomer": {"astro-ui": default_probes}},
+    "charts/astronomer/templates/cli-install/cli-install-deployment.yaml": {"astronomer": {"cli-install": default_probes}},
+    "charts/astronomer/templates/commander/commander-deployment.yaml": {"astronomer": {"commander": default_probes}},
+    "charts/astronomer/templates/houston/api/houston-deployment.yaml": {"astronomer": {"houston": default_probes}},
+    "charts/astronomer/templates/houston/worker/houston-worker-deployment.yaml": {
+        "astronomer": {"houston": {"worker": default_probes}}
+    },
+    "charts/astronomer/templates/registry/registry-statefulset.yaml": {"astronomer": {"registry": default_probes}},
+    "charts/elasticsearch/templates/client/es-client-deployment.yaml": {"elasticsearch": {"client": default_probes}},
+    "charts/elasticsearch/templates/data/es-data-statefulset.yaml": {"elasticsearch": {"data": default_probes}},
+    "charts/elasticsearch/templates/exporter/es-exporter-deployment.yaml": {"elasticsearch": {"exporter": default_probes}},
+    "charts/elasticsearch/templates/master/es-master-statefulset.yaml": {"elasticsearch": {"master": default_probes}},
+    "charts/elasticsearch/templates/nginx/nginx-es-deployment.yaml": {"elasticsearch": {"nginx-es": default_probes}},
+    "charts/external-es-proxy/templates/external-es-proxy-deployment.yaml": {"external-es-proxy": default_probes},
+    "charts/fluentd/templates/fluentd-daemonset.yaml": {"fluentd": default_probes},
+    "charts/grafana/templates/grafana-deployment.yaml": {"grafana": default_probes},
+    "charts/kibana/templates/kibana-deployment.yaml": {"kibana": default_probes},
+    "charts/kube-state/templates/kube-state-deployment.yaml": {"kube-state": default_probes},
+    "charts/nats/templates/statefulset.yaml": {"nats": default_probes},
+    "charts/nginx/templates/nginx-deployment-default.yaml": {"nginx": default_probes},
+    "charts/nginx/templates/nginx-deployment.yaml": {"nginx": default_probes},
+    "charts/pgbouncer/templates/pgbouncer-deployment.yaml": {"pgbouncer": default_probes},
+    "charts/postgresql/templates/statefulset-slaves.yaml": {"postgres": default_probes},
+    "charts/postgresql/templates/statefulset.yaml": {"postgres": default_probes},
+    "charts/prometheus/templates/prometheus-statefulset.yaml": {"prometheus": default_probes},
+    "charts/prometheus-blackbox-exporter/templates/deployment.yaml": {"prometheus-blackbox-exporter": default_probes},
+    "charts/prometheus-node-exporter/templates/daemonset.yaml": {"prometheus-node-exporter": default_probes},
+    "charts/prometheus-postgres-exporter/templates/deployment.yaml": {"prometheus-postgres-exporter": default_probes},
+    "charts/stan/templates/statefulset.yaml": {"stan": default_probes},
+}
```

Comment:
@pgvishnuram @shubh-am-kumar what do you two think of this approach? This seems like the only way we'll be able to get a comprehensive test of all the different ways we will need to configure probes, due to the inconsistencies in our chart. This seems to be working really well, but I'm curious if you think it's a bad idea.

Reply:
@danielhoherd you've tagged the wrong Shubham :)
cc: @Shubham-astro